src/lib/Scenes/Fair/Screens/FairDetail.tsx

```diff
@@ -206,8 +206,11 @@ export const FairDetailContainer = createPaginationContainer(
             cursor
             node {
               ...FairBoothPreview_show
-              ...FairBooth_show
               ...ShowArtworks_show
+              ...FairBoothHeader_show
```

Comment:
We shouldn't need any of those fragments right? these come from `FairBooth_show` which is the unneeded fragment we want to eliminate from the initial query